platforms/posix/src/px4_layer/drv_hrt.cpp

```diff
@@ -162,7 +162,11 @@ hrt_abstime _hrt_absolute_time_internal()
 {
 	struct timespec ts;
 	px4_clock_gettime(CLOCK_MONOTONIC, &ts);
+#ifdef __PX4_QURT
+	return ts_to_abstime(&ts) + dsp_offset;
+#else
 	return ts_to_abstime(&ts);
+#endif
 }
 
 #ifdef __PX4_QURT
```

Comment:
Super nitpicky, can you fix this whitespace while you're at it?

Reply:
No problem. Did I forgot to setup git hooks again, or am I thinking about different project ?

Reply:
Seems to slip through astyle, or at least the way we have it configured.